Hero.cpp

```diff
@@ -45,9 +46,10 @@ void Hero::Attack(Monster &enemy)
             LevelUp();
         }
     }
-    if (damage - enemy.getDefense() > 0)
+    if (damage.physical - enemy.getDefense() > 0)
```

Comment:
Ha physical sebzést teljesen levédi az áldozat, attól még a magical damage az még beüt, szóval kap érte xp-t is